components/03-modules/product-view/product-review/product-review.hbs

```diff
@@ -0,0 +1,21 @@
+<section class="product-review">
+    <h2 class="product-review__title">
+        {{ title }}
+    </h2>
+    <div class="product-review__summary">
+        {{ render '@review--summary' summary merge=true }}
+    </div>
+    <div class="product-review__reviews">
+        <ul class="list">
+            {{#each reviews }}
+                <li class="list__item">
+                    {{ render '@review--default' this }}
+                </li>
+            {{/each}}
+        </ul>
+    </div>
+    <h2 class="product-review__heading">
+        You're reviewing:
```

Comment:
you can move this to variable

Reply:
I remove this, it's already in `review--add` component, my bad. Fixed